ckanext/switzerland/helpers/date_helpers.py

```diff
@@ -120,6 +120,19 @@ def display_if_datetime(value):
         return None
 
 
+def display_if_other_formats(value):
+    """dates/datetime values with other formats will be displayed in
+    ckanext.switzerland.date_picker_format
+    """
+    try:
+        for date_format in ALLOWED_DATE_FORMATS:
+            dt = datetime.strptime(value, date_format)
+            if isinstance(dt, datetime):
+                return value
```

Comment:
hmm, maybe I don't understand this enough, but do we want to display the value? or the datetime we get out of it? I prefer siso but I am not sure in this case. What do you think?